java2cpp.y

```diff
@@ -93,9 +93,16 @@ RETURN_ST 	: RETURN { printf("return "); } EXPRESION SEMICOLON { printf(";\n");
 BREAK_ST	: BREAK SEMICOLON  { printf("break;\n"); }
 			;
 
-STDIO	: PRINTLN { printf("std::cout"); } LP { printf(" << "); } EXPRESION RP { printf(" <<  std::endl"); } SEMICOLON { printf(";\n"); }
+STDIO	        : PRINTLN { printf("std::cout"); } LP { printf(" << "); } EXPRESION RP { printf(" <<  std::endl"); } SEMICOLON { printf(";\n"); }
 		| PRINT { printf("std::cout"); } LP { printf(" << "); } EXPRESION RP SEMICOLON { printf(";\n"); }
 		;
+		
+SCANNER_OBJECT : SCANNER { printf("string"); } VAR { printf("%s;", yylval.var_name); } HAS_ASSIGNMENT NEW SCANNER {printf("std::cin");} LP SYS.IN RP {printf(">>");} VAR SEMICOLON { printf("%s;", yylval.var_name); } 
+               ;
+
+
+MY_INPUT        : VAR ASSIGNMENT NEW SCANNER {printf("std::cin");} LP SYS.IN RP {printf(">>");} VAR  SEMICOLON { printf("%s;", yylval.var_name); } 
```

Comment:
Remove the last ```VAR```

It would look like this:
```yacc
MY_INPUT        : VAR ASSIGNMENT NEW SCANNER {printf("std::cin");} LP SYS_IN RP {printf(">>");} SEMICOLON { printf("%s;", yylval.var_name); }
                ;
```